### PR TITLE
Load less file in debug mode

### PR DESCRIPTION
**Pull request by fredj**

### CODE REVIEW

**Comment by sbrunner** [comment]
Looks good :-)

**Comment by fredj** [comment]
the path to `less.min.js` is probably wrong (`node_modules/less/dist/less.min.js`). `request.static_url` should be used

**Comment by sbrunner** [comment]
Right, probably with node_modules (https://github.com/camptocamp/c2cgeoportal/blob/master/c2cgeoportal/__init__.py#L278)

File: c2cgeoportal/scaffolds/update/CONST_Makefile_tmpl [hunk 1]
```diff
@@ -498,10 +497,6 @@ $(APP_OUTPUT_DIR)/build.min.css: $(LESS_FILES) .build/node_modules.timestamp
 	mkdir -p $(dir $@)
 	./node_modules/.bin/lessc --clean-css $(PACKAGE)/static-ngeo/less/$(PACKAGE).less $@
 
-$(APP_OUTPUT_DIR)/build.css: $(LESS_FILES) .build/node_modules.timestamp
-	mkdir -p $(dir $@)
-	./node_modules/.bin/lessc $(PACKAGE)/static-ngeo/less/$(PACKAGE).less $@
-
```
Comment (sbrunner) [inline c2cgeoportal/scaffolds/update/CONST_Makefile_tmpl]: Just a remark, to be as we are with `cgxp` we shouldn't use the `--clean-css` when `DEVELOPMENT` is set to `TRUE`...

**Comment by fredj** [comment]
`${request.static_url('epfl_authgeoportail:node_modules/less/dist/less.js')}` does not work

**Comment by sbrunner** [comment]
I think that should be:
`${request.static_url('node_modules:less/dist/less.js')}`

**Comment by sbrunner** [comment]
no I'm wrong ...

File: c2cgeoportal/scaffolds/update/CONST_Makefile_tmpl (hunk 1 above)
Comment (fredj) [inline c2cgeoportal/scaffolds/update/CONST_Makefile_tmpl]: Fixed with #1614

**Comment by fredj** [comment]
It's working now ...

**Comment by sbrunner** [comment]
Looks good for me :-)

**Pull request merged**